src/streaming/utils/Capabilities.js

```diff
@@ -187,8 +187,10 @@ function Capabilities() {
      * @return {boolean}
      */
     function supportsEssentialProperty(ep) {
+        let supportedEssentialProps = settings.get().streaming.capabilities.supportedEssentialProperties;
+        supportedEssentialProps.push(...THUMBNAILS_SCHEME_ID_URIS);
         try {
-            return THUMBNAILS_SCHEME_ID_URIS.indexOf(ep.schemeIdUri) !== -1;
+            return supportedEssentialProps.indexOf(ep.schemeIdUri) !== -1;
```

Comment:
For discussion: I think it would be cleaner to add the schemeIds for thumbnails to the default settings. If for some reason an application/user wants to remove it from the list, it can be done via the settings then.

Reply:
Agree.

Let me check any potential implications of such a change.

Reply:
I checked this..

additionally to this instance you commented on here, `THUMBNAILS_SCHEME_ID_URIS` is also used in https://github.com/Dash-Industry-Forum/dash.js/blob/124920530d8503bd7581e1ffed300823b54c2512/src/streaming/thumbnail/ThumbnailTracks.js#L142

`Capabilities.js` uses _all_ essential schemeIds and consequently the `Array.indexOf`-method can be based on the array from `Settings.js`.

In contrast, `ThumbnailTracks.js` is restricted to those two thumbnail-related schemeIds only.

With this, I see some options:

1. Include the following two lines in `Settings.js`  https://github.com/Dash-Industry-Forum/dash.js/blob/124920530d8503bd7581e1ffed300823b54c2512/src/streaming/thumbnail/ThumbnailTracks.js#L42 
2. Hard-code above mentioned schemeIds in Settings.js _in parallel to_ the existing definition in `ThumbnailTracks.js`

Both options come with their pros/cons. 
@dsilhavy please advice

Reply:
Thanks for checking this. I suggest we add the schemeIds for thumbnails to `Constants.js`. Similar to the ones for DVB low latency etc. Then we import from there to all the related classes. This would require changes in all related files, though not only `Settings.js`. 
What do you think?

Reply:
OK, will try this now. Standby...

Reply:
Done.
Please review https://github.com/Dash-Industry-Forum/dash.js/pull/4385/commits/4813c8a4c43317aaeaf642ce4886423a6ac14123

Reply:
lgtm than you :)